### PR TITLE
add mesa/libGLU

### PR DESCRIPTION
**Pull request by bgruening**

### CODE REVIEW

File: recipes/libglu/build.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+export CFLAGS="-I${PREFIX}/include"
+export LDFLAGS="-L${PREFIX}/lib"
+./configure  --prefix=$PREFIX ##--disable-static
+make
+make install
```
Comment (ocefpaf) [inline recipes/libglu/build.sh]: Is there a `make check` that we can run?

Comment (bgruening) [inline recipes/libglu/build.sh]: No check, and no `make test`. Sorry.

Comment (ocefpaf) [inline recipes/libglu/build.sh]: No problem. (Had to ask in case you forgot to add it.)

Comment (bgruening) [inline recipes/libglu/build.sh]: I appreciate that! Thanks!

File: recipes/libglu/bld.bat [hunk 2]
```diff
@@ -0,0 +1,18 @@
+mkdir build
+cd build
+
+:: Configure.
```
Comment (bgruening) [inline recipes/libglu/bld.bat]: @ocefpaf I have no clue about the Windows build and the syntax. Are static builds supported? Should I just skip it in win?

Comment (ocefpaf) [inline recipes/libglu/bld.bat]: You are asking the wrong guy. I would skip it for now, get this into a feedstock, and then ask for help there. That way you get `Linux` and `OS X` out of the door and can start using it.

Comment (bgruening) [inline recipes/libglu/bld.bat]: Love this answer!

**Pull request merged**